File: atest.cs
using static System.Console;
class atest
{
public static void Main()
{

  int flag = 1;
  string write;
  while (  flag ==  1 ){
    WriteLine(   "Write Something:");
    write = ReadLine();
    WriteLine(   "You typed:" + write);
    if (  write ==     "q"){
      flag=0;
    }
  }
  WriteLine(   "Quit Program");

}}

File: atest.cpp
#include <iostream>
#include <string>
using namespace std;
int main()
{
  
  int flag = 1;
  std::string write;
  while (  flag ==  1 ){
    std::cout << "Write Something:\n";
    std::cin >> write;
    std::cout << "You typed:"<< write <<" \n";
    if (  write ==     "q"){
      flag=0;
    }
  }
  std::cout << "Quit Program";
  return 0;
}

